test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -43,4 +42,30 @@ private void assertAreUnique(Object... objects) {
     private void assertNotUnique(Object... objects) {
         assertFalse(Utils.elementsAreUnique(Arrays.asList(objects)));
     }
+
+    private void assertIsNull(Object... objects) {
+        assertTrue(Utils.isAnyNull(objects));
+    }
+
+    private void assertIsNotNull(Object... objects) {
+        assertFalse(Utils.isAnyNull(objects));
+    }
+
+    @Test
+    public void isAnyNull() throws Exception {
+        //empty
+        assertIsNotNull();
+
+        //One item only
+        assertIsNotNull("abc");
+
+        //Two items
+        assertIsNotNull("abc", "def");
+        assertIsNull("abc", null);
+        assertIsNull(null, "abc");
+
+        //No object
+        assertIsNull((Object) null);
+
```

Comment:
Could add a few more test cases for checking:
- Multiple null items
- Multiple non-null items
- Combinations of null and non-null item**'s'**